test/dartdoc_test_base.dart

```diff
@@ -40,7 +40,7 @@ abstract class DartdocTestBase {
 
   String get sdkConstraint => '>=3.3.0 <4.0.0';
 
-  List<String> get experiments => [];
+  List<String> get experiments => ['wildcards'];
```

Comment:
The experiment is named `wildcard-variables` actually :D

Reply:
Oops! I don't see an easy way to verify that analyzer / `dart` is not reporting an error. I should be able to wire up an error listener, primarily useful for tests like this. But I think a task for another day.